getting-started.udon

```diff
@@ -7,36 +7,31 @@
 
 # Getting Started
 
-Pages in this section are guides to getting you started with Urbit.
+Welcome to Urbit, a personal server that makes you the owner of your digital life. Urbit isn't ready for serious use yet, but it definitely works. Follow this step-by-step guide to get an Azimuth point, install Urbit, and join the community.
```

Comment:
This feels a little off to me still.

I wonder if the first sentence should stay vague. "Makes you the owner of your digital life" sets up the expectation that there's actually software in existence to do that. Right now Urbit doesn't actually provide that experience. How about just "Welcome to Urbit, a personal server."

"Urbit isn't ready for serious use yet, but it definitely works" also worries me – I'm not sure if we can say "definitely works" and mean it. "Urbit isn't ready for serious use yet, but if you're curious you can try it out. Follow this step-by-step guide to get an Azimuth point, install Urbit, and talk to other users on the network."

Reply:
How about "but it works as alpha-stage software."

Reply:
We are [pre-alpha](https://en.wikipedia.org/wiki/Software_release_life_cycle#Pre-alpha), but I wouldn't use this terminology at all.

Reply:
I think this is a matter of debate, and "alpha" properly carries the connotation of "rudimentary"

Reply:
Alright, I am just going to drop the terminology